include/attributes_builder.h

```diff
@@ -85,6 +88,10 @@ class AttributesBuilder {
     }
   }
 
+  // If key suffixed with ".ip", try to convert its value to ipv4 or ipv6.
```

Comment:
Can you add a comment that this function is only needed to support legacy mixerclient config format? The v2 attribute format is strongly typed and things like IP are already passed as `[]byte`.

Reply:
Done